app/build.gradle

```diff
@@ -353,7 +353,7 @@ distributions {
                 exclude 'es.jar'
                 exclude 'es-*.jar'
             }
-            into("lib") {
+            into("lib/main") {
```

Comment:
the recent change in the packaging should actually be able to deal with that change since we recursively copy the whole lib folder.